src/feedback-screen.tsx

```diff
@@ -148,14 +155,19 @@ interface FeedbackProps {
     description?: string | Array<string>;
     children?: React.ReactNode;
     unstable_inlineInDesktop?: boolean;
+    dataAttributes?: DataAttributes;
+}
+
+interface AssetFeedbackProps extends FeedbackProps {
+    imageUrl?: string;
+    imageFit?: 'fit' | 'fill'; // defaults to 'fill'
```

Comment:
Perhaps `'contain'` and `'cover'` would be better values for this prop. At least the name explains what you get in css.

Reply:
I used the names that are already being used in HighlightedCards. I think DesignCore members use that naming.

Reply:
ok then, let's keep naming